src/test/unit/parsers/test_utils.py

```diff
@@ -261,6 +261,30 @@ def test_recursive_json_join(d1, d2, result):
             },
             {"triggeredBy": "unknown"},
         ),
+        (  # API GW V2
+            {
+                "version": "2.0",
+                "headers": {},
+                "requestContext": {
+                    "domainName": "r3pmxmplak.execute-api.us-east-2.amazonaws.com",
+                    "domainPrefix": "r3pmxmplak",
+                    "http": {
+                        "method": "GET",
+                        "path": "/default/nodejs-apig-function-1G3XMPLZXVXYI",
+                    },
+                    "requestId": "JKJaXmPLvHcESHA=",
+                    "stage": "default",
+                },
+            },
+            {
+                "triggeredBy": "apigw",
```

Comment:
It looks like we're missing information here - 
maybe we want to change to `apigw2`?

Reply:
For the backend, it's regular API-gw.
Just other places to look for our interesting fields